tests/Integration/Integration.Tests/ShortagesReportModuleTests/WhenGettingReportNoResults.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace Linn.Purchasing.Integration.Tests.ShortagesReportModuleTests
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+
+    using FluentAssertions;
+
+    using Linn.Common.Reporting.Models;
+    using Linn.Common.Reporting.Resources.ReportResultResources;
+    using Linn.Purchasing.Integration.Tests.Extensions;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenGettingReportNoResults : ContextBase
```

Comment:
I don't really think you need an extra carter/integration test for this scenario since its only your domain service you've changed, which is mocked out anyway

Reply:
No worries, will remove then. Thanks for the reminder!

Reply:
if you really felt the urge to test the no results scenario you could do a unit test for your domain service instead